app/jenkins-for-jira-ui/src/components/ConnectionPanel/ConnectionPanel.styles.tsx

```diff
@@ -6,7 +6,7 @@ export const connectionPanelContainer = css`
 	box-shadow: 0px 2px 4px 0px #091E4240;
 	display: flex;
 	flex-direction: column;
-	padding: ${token('space.300')};
+	padding: ${token('space.300')} ${token('space.300')} ${token('space.0')};
```

Comment:
Needed to fix the padding at the bottom of the screen and figured this PR was small enough that I could squeeze it in. Goal is that this PR is the last of the UI + content (until I get more) for the admin page.